modules/nw-mutual-tls-auth.adoc

```diff
@@ -1,31 +1,40 @@
 // Module included in the following assemblies:
 //
-// * ingress/ingress-operator.adoc
+// * networking/ingress-operator.adoc
 
 :_content-type: PROCEDURE
 [id=nw-mutual-tls-auth_{context}]
 = Configuring mutual TLS authentication
 
-You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` value configures the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map. The config map contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate. Optionally, you can configure a list of certificate subject filters.
+You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` value configures the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map. The config map contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate. Optionally, you can also configure a list of certificate subject filters.
 
 If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Operator downloads the CRL and configures the Ingress Controller to acknowledge it. Requests that do not provide valid certificates are rejected.
 
 .Prerequisites
 
 * You have access to the cluster as a user with the `cluster-admin` role.
+* You have a PEM-encoded CA certificate bundle.
+* If your CA bundle references a CRL distribution point, you must also add a certificate with the HTTP URI specifying the CRL for that CA. For example:
```

Comment:
This is confusing, a certificate signed by the CA or Sub CA will have the CRL distribution point, so the user must include the end-entity or leaf certificate to the clientCA bundle. This certificate needs to include a HTTP URI under CRL Distribution Points, as described in rfc5280.